packages/cozy-harvest-lib/src/helpers/oauth.js

```diff
@@ -128,6 +129,11 @@ export const getOAuthUrl = ({
   return oAuthUrl
 }
 
+const getAppSlug = () => {
+  const root = document.querySelector('[role=application]')
+  return get(root, 'dataset.cozyAppSlug')
```

Comment:
Querying the DOM like that involves that we only use Harvest within a DOM served by the cozy-stack. 

If we plan to use it in our next project, client side, we should not rely on that fact, right? 

Since we're in a context with a CozyClient instance https://github.com/cozy/cozy-libs/blob/2e552961595a1cab8d593cc9e014ca0d95f1b68e/packages/cozy-harvest-lib/src/helpers/oauth.js#L147 we can get the current application by reading client.appMetadata.slug

Reply:
Nice! Thanks!